CppCoreGuidelines.md

```diff
@@ -4516,6 +4525,39 @@ This is known as "the rule of five" or "the rule of six", depending on whether y
 If you want a default implementation of a default operation (while defining another), write `=default` to show you're doing so intentionally for that function.
 If you don't want a default operation, suppress it with `=delete`.
 
+##### Example, good
+
+When a destructor needs to be declared just to make it `virtual`, it can be
+defined as defaulted. To avoid suppressing the implicit move operations they
+they must also be declared, and then to avoid the class becoming move-only
```

Comment:
here is an accidential doubling of "they"

Reply:
Fixed, thanks.